src/desktop/apps/article/components/App.tsx

```diff
@@ -7,15 +7,45 @@ import { EditButton } from "desktop/apps/article/components/EditButton"
 import { ArticleLayout } from "./layouts/Article"
 import { data as sd } from "sharify"
 import { ArticleProps } from "@artsy/reaction/dist/Components/Publishing/Article"
+import { isProduction } from "lib/environment"
+import { articleAdCategory } from "../ad_category"
 
 export interface AppProps extends ArticleProps {
   templates?: {
     SuperArticleFooter: string
     SuperArticleHeader: string
   }
 }
-
+interface HtlbidProps {
+  cmd: any
+  setTargeting?: (a: string, b: string) => any
+}
 export class App extends React.Component<AppProps> {
+  componentDidMount() {
+    this.mountDFPScript()
+  }
+
+  mountDFPScript() {
+    const { article } = this.props
+    let htlbid: HtlbidProps = {
+      cmd: [],
+    }
+    let testingState = isProduction ? "no" : "yes"
+    let pageType = articleAdCategory(article)
+    let postID = article.id
```

Comment:
if these variables aren't reassigned then `const` is preferred over `let` (the linter should also catch this).